### PR TITLE
Update "Salesforce Rest API Connector Example" Doc

### PR DESCRIPTION
**Pull request by chathuranga-jayanath-99**
## Purpose
This PR includes updates to the below mentioned page.
https://mi.docs.wso2.com/en/4.3.0/reference/connectors/salesforce-connectors/sf-rest-connector-example/

The updated page will appear as shown in the attached PDF below.
[Salesforce REST API Example - WSO2 Micro Integrator Documentation 4.3.0.pdf](https://github.com/user-attachments/files/16277793/Salesforce.REST.API.Example.-.WSO2.Micro.Integrator.Documentation.4.3.0.pdf)

### CODE REVIEW

File: en/docs/reference/connectors/salesforce-connectors/sf-rest-connector-example.md [hunk 1]
```diff
@@ -21,49 +21,42 @@ The user calls the Salesforce REST API. It invokes the **create** sequence and c
...
-## Configure the connector in WSO2 Integration Studio
-
-Connectors can be added to integration flows in [WSO2 Integration Studio](https://wso2.com/integration/integration-studio/). Once added, the operations of the connector can be dragged onto your canvas and added to your sequences.
-
-### Import the connector
-
-Follow these steps to set up the Integration Project and the Connector Exporter Project. 
-
-{!includes/reference/connectors/importing-connector-to-integration-studio.md!} 
+## Setting up the Integration Project
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/salesforce-connectors/sf-rest-connector-example.md:24]:
```suggestion
## Set up the integration project
```

File: en/docs/reference/connectors/salesforce-connectors/sf-rest-connector-example.md [hunk 2]
```diff
@@ -21,49 +21,42 @@ The user calls the Salesforce REST API. It invokes the **create** sequence and c
...
+
+1. Follow the steps in [create integration project]({{base_path}}/develop/create-integration-project/) guide to set up the Integration Project. 
+
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/salesforce-connectors/sf-rest-connector-example.md:27]: remove extra new line

File: en/docs/reference/connectors/salesforce-connectors/sf-rest-connector-example.md [hunk 3]
```diff
@@ -21,49 +21,42 @@ The user calls the Salesforce REST API. It invokes the **create** sequence and c
...
+2. Create a new Salesforcerest connection. 
+    1. Goto `Local Entries` -> `Connections` and click on the `+` sign. 
+    2. Select  `Salesforcerest` connector.
+        <img src="{{base_path}}/assets/img/integrate/connectors/sf-rest-conn-add-connection.png" title="Add new salesforce connection" width="800" alt="Add new salesforce connection"/>
+
+    3. Use following values can to create the connection. 
+        - Connection Name - SalesforceRestConnection
+        - Connection Type - init
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/salesforce-connectors/sf-rest-connector-example.md:35]:
```suggestion
        - Connection Type - `init` 
```

File: en/docs/reference/connectors/salesforce-connectors/sf-rest-connector-example.md [hunk 4]
```diff
@@ -21,49 +21,42 @@ The user calls the Salesforce REST API. It invokes the **create** sequence and c
...
+
+1. Follow the steps in [create integration project]({{base_path}}/develop/create-integration-project/) guide to set up the Integration Project. 
+
+2. Create a new Salesforcerest connection. 
+    1. Goto `Local Entries` -> `Connections` and click on the `+` sign. 
+    2. Select  `Salesforcerest` connector.
+        <img src="{{base_path}}/assets/img/integrate/connectors/sf-rest-conn-add-connection.png" title="Add new salesforce connection" width="800" alt="Add new salesforce connection"/>
+
+    3. Use following values can to create the connection. 
+        - Connection Name - SalesforceRestConnection
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/salesforce-connectors/sf-rest-connector-example.md:34]:
```suggestion
        - Connection Name - `SalesforceRestConnection`
```

File: en/docs/reference/connectors/salesforce-connectors/sf-rest-connector-example.md [hunk 5]
```diff
@@ -21,49 +21,42 @@ The user calls the Salesforce REST API. It invokes the **create** sequence and c
...
-
-Follow these steps to set up the Integration Project and the Connector Exporter Project. 
-
-{!includes/reference/connectors/importing-connector-to-integration-studio.md!} 
+## Setting up the Integration Project
+
+1. Follow the steps in [create integration project]({{base_path}}/develop/create-integration-project/) guide to set up the Integration Project. 
+
+2. Create a new Salesforcerest connection. 
+    1. Goto `Local Entries` -> `Connections` and click on the `+` sign.
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/salesforce-connectors/sf-rest-connector-example.md:29]:
```suggestion
    1. Go to `Local Entries` -> `Connections` and click on the `+` sign. 
```

File: en/docs/reference/connectors/salesforce-connectors/sf-rest-connector-example.md [hunk 6]
```diff
@@ -21,49 +21,42 @@ The user calls the Salesforce REST API. It invokes the **create** sequence and c
...
-        - **accessToken** : Value of the access token to access the API via request.
-        - **refreshToken** : Value of the refresh token.
-       
-        <img src="{{base_path}}/assets/img/integrate/connectors/salesforce-api-init-operation-sequnce1.png" title="Add values to the init operation" width="800" alt="Add values to the init operation"/>
-    
+
+1. Provide `create` as name and create the sequence. 
      
-2. Set up the created operation.
+2. Set up the create operation.
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/salesforce-connectors/sf-rest-connector-example.md:59]:
```suggestion
2. Set up the `create` operation.
```

File: en/docs/reference/connectors/salesforce-connectors/sf-rest-connector-example.md [hunk 7]
```diff
@@ -72,95 +65,95 @@ Now follow the steps below to add configurations to the sequence.
...
-    
-    3. To get the input values in to the API we can use the [property mediator]({{base_path}}/reference/mediators/property-mediator/). Navigate into the **Palette** pane and select the graphical mediators icons listed under **Mediators** section. Then drag and drop the `Property` mediators into the Design pane as shown bellow.
+    2. Select the created `create` sequence. 
+    3. Add the `create` operation from the `Salesforcerest` connector. 
+
+        <img src="{{base_path}}/assets/img/integrate/connectors/sf-rest-conn-add-create-operation.png" title="Add create operation" width="800" alt="Add create operations"/>
     
-        <img src="{{base_path}}/assets/img/integrate/connectors/salesforce-api-drag-and-drop-property-mediator.png" title="Add property mediators" width="800" alt="Add property mediators"/>
+    3. To get the input values in to the API we can use the [property mediator]({{base_path}}/reference/mediators/property-mediator/). Add the `Property` mediators into the Design pane as shown bellow.
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/salesforce-connectors/sf-rest-connector-example.md:73]:
```suggestion
    3. To get the input values in to the API we can use the [property mediator]({{base_path}}/reference/mediators/property-mediator/). Add the `Property` mediators into the design pane as shown bellow.
```

File: en/docs/reference/connectors/salesforce-connectors/sf-rest-connector-example.md [hunk 8]
```diff
@@ -21,146 +19,128 @@ The user calls the Salesforce REST API. It invokes the **create** sequence and c
...
-{!includes/reference/connectors/importing-connector-to-integration-studio.md!} 
+## Set up the integration project
+1. Follow the steps in [create integration project]({{base_path}}/develop/create-integration-project/) guide to set up the Integration Project. 
+2. Create a new `Salesforcerest` connection. 
+    1. Go to `Local Entries` -> `Connections` and click on the `+` sign.
+    2. Select  `Salesforcerest` connector.
+        <img src="{{base_path}}/assets/img/integrate/connectors/sf-rest-conn-add-connection.png" title="Add new salesforce connection" width="800" alt="Add new salesforce connection"/>
+
+    3. Use following values can to create the connection. 
+        - Connection Name - `SalesforceRestConnection`
```
Comment (chanikag) [inline en/docs/reference/connectors/salesforce-connectors/sf-rest-connector-example.md:30]: Make the property names bold

File: en/docs/reference/connectors/salesforce-connectors/sf-rest-connector-example.md [hunk 9]
```diff
@@ -251,8 +227,119 @@ Invoke the API as shown below using the curl command. Curl application can be do
...
+                "url": "/services/data/v44.0/sobjects/Account/001dM00000HemdeQAB"
+            },
+            "Id": "001dM00000HemdeQAB",
+            "Name": "Sample Account for Entitlements"
+        }
+    ]
+}
+```
 
 ## What's Next
```
Comment (chanikag) [inline en/docs/reference/connectors/salesforce-connectors/sf-rest-connector-example.md:344]:
```suggestion
## What's next
```

File: en/docs/reference/connectors/salesforce-connectors/sf-rest-connector-example.md [hunk 10]
```diff
@@ -215,23 +203,11 @@ You can download the ZIP file and extract the contents to get the project code.
     You may need to update the value of the access token and make other such changes before deploying and running this project.
 
 ## Deployment
+To deploy and run the project, please refer to the [build and run]({{base_path}}/develop/deploy-artifacts/#build-and-run) guide.
 
-Follow these steps to deploy the exported CApp in the integration runtime. 
-
-**Deploying on Micro Integrator**
+You can further refer the application deployed through the CLI tool. See the instructions on [managing integrations from the CLI]({{base_path}}/observe-and-manage/managing-integrations-with-micli).
```
Comment (chanikag) [inline en/docs/reference/connectors/salesforce-connectors/sf-rest-connector-example.md:208]:
```suggestion
You can further refer to the application deployed through the CLI tool. See the instructions on [managing integrations from the CLI]({{base_path}}/observe-and-manage/managing-integrations-with-micli). 
```

**Review by chanikag — APPROVED** [review]

**Pull request merged**